_partials/_prometheus-integrate.md

```diff
@@ -0,0 +1,95 @@
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
+
+[Prometheus][prometheus] is an open-source monitoring system with a dimensional data model, flexible query language, and a modern alerting approach.
+
+This page shows you how to export your $SERVICE_SHORT telemetry to Prometheus using [PostgreSQL Exporter][postgresql-exporter].
+
+## Prerequisites
+
+<IntegrationPrereqs />
+
+- [Download and run Prometheus][install-prometheus].
+- [Install PostgreSQL Exporter][install-exporter].
+
+To reduce latency and potential data transfer costs, install Prometheus and PostgreSQL Exporter on a machine in the 
+same AWS region as your $SERVICE_LONG.
+
+## Export $SERVICE_SHORT telemetry to Prometheus
+
+To export your data, do the following:
+
+<Procedure>
+
+1. **Create a user to access metrics data on your $SERVICE_LONG**
+
+   1. Connect to your $SERVICE_LONG:
+
+      For $CLOUD_LONG, open an [SQL editor][run-queries] in [$CONSOLE][open-console]. For self-hosted, use [`psql`][psql].  
+
+   1. Create a user named `monitoring` with a password:
+
+      ```sql
+      CREATE USER monitoring WITH PASSWORD '<password>';
+      ```
+
+   1. Grant the `pg_read_all_stats` permission to the `monitoring` user:
+
+      ```sql
+      GRANT pg_read_all_stats to monitoring;
+      ```
+
+1. **Export metrics data using PostgreSQL Exporter**
+
+   1. Configure PostgreSQL Exporter to connect to your $SERVICE_LONG:
+
+      Update the `Service URL:` in your [connection details][connection-info] to connect to your $SERVICE_LONG as the 
```

Comment:
@billy-the-fish This is confusing: 
1. There is no <service URL> placeholder in the variable below. 
2. You don't update the service URL **in** your connection details. You take it **from** your connection details and update it **in** the variable below. 
3. This variable is configured only if you use binaries. If you install with Docker, it's done differently

Reply:
4. This is how this variable is supposedly set fro Docker: 
```
docker run -d \
  -e DATA_SOURCE_NAME="postgresql://username:password@host:port/dbname?sslmode=disable" \
  -p 9187:9187 \
  prometheuscommunity/postgres-exporter
```

Reply:
Added